collections/src/main/scala/strawman/collection/mutable/Builder.scala

```diff
@@ -87,10 +88,10 @@ class StringBuilder(private val sb: java.lang.StringBuilder) extends Builder[Cha
   // Methods required to make this an IndexedSeq:
   def apply(i: Int): Char = sb.charAt(i)
   def iterableFactory: strawman.collection.SeqFactory[IndexedSeq] = IndexedSeq
-  protected[this] def fromSpecificIterable(coll: strawman.collection.Iterable[Char]): IndexedSeq[Char] =
-    iterableFactory.from(coll)
-  protected[this] def newSpecificBuilder(): strawman.collection.mutable.Builder[Char, IndexedSeq[Char]] =
-    iterableFactory.newBuilder()
+  protected[this] def fromSpecificIterable(coll: strawman.collection.Iterable[Char]): StringBuilder =
+    new StringBuilder() ++= coll
+  protected[this] def newSpecificBuilder(): strawman.collection.mutable.Builder[Char, StringBuilder] =
+    IndexedSeq.newBuilder[Char]().mapResult(r => new StringBuilder() ++= r)
```

Comment:
should this be more efficient?

Reply:
Maybe `IndexedSeq.newBuilder[Char]().mapResult(r => new StringBuilder(r))`.

Reply:
I'd use a `GrowableBuilder` (like in 2.12) so that `StringBuilder` can build itself without copying on `result()`